internal/install/_static/Dockerfile.terraform_deployer

```diff
@@ -1,6 +1,23 @@
-FROM hashicorp/terraform:light
-ENV TF_IN_AUTOMATION=true
+FROM hashicorp/terraform:light as terraform
+
+FROM python:3-alpine
+
+# required by gcloud SDK
+RUN apk add curl
```

Comment:
nit: `apk add --no-cache`

Reply:
also:

I checked [here](https://hub.docker.com/layers/hashicorp/terraform/light/images/sha256-ea87aec24e5c37af99d2b73c4c52d4b17a2e9ea9582d68a2f8a7c2968cd93ca5?context=explore) and there is also Git and OpenSSH installed in the original Docker image. Let's add them also to this custom one.